src/lwc/geFormWidget/geFormWidget.js

```diff
@@ -1,5 +1,8 @@
-import { LightningElement, api } from 'lwc';
-import {hasNestedProperty, isUndefined} from 'c/utilCommon';
+import { LightningElement, api, track } from 'lwc';
+import {apiNameFor, getSubsetObject, isUndefined} from 'c/utilCommon';
+
+import DI_ADDITIONAL_OBJECT_JSON_FIELD from '@salesforce/schema/DataImport__c.Additional_Object_JSON__c';
+import DI_DONATION_AMOUNT_FIELD from '@salesforce/schema/DataImport__c.Donation_Amount__c';
```

Comment:
It feels a little weird that  these imports are here, instead of in the GAU widget specifically. But I'm assuming this is all part of the grand plan!

Reply:
I think you're right @ptewson-sfdo , The reason why those are imported there is because we're slicing out the data we need from formState for the gau widget in `geFormWidget`, I think `geFormWidgetAllocation` should handle slicing out the data it needs from `formState`. Thoughts? @dfuller114 and @kenlewis-sfdo

Reply:
This might be a good candidate to wait on resolving regardless, if the change would jeopardize the work closing  this sprint

Reply:
Yeah, I would support pushing the allocation specific imports down into the Allocation widget, but also think this should stay as is for now and we can take a breath and look at the shape and hierarchy of the Widget + Allocation stuff after this work is in and then maybe do some analysis and design around our larger widget implementation and what we want to do going forward.